soc/arm/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -293,6 +293,7 @@ config SOC_NRF52833
 	select HAS_HW_NRF_RADIO_BLE_2M
 	select HAS_HW_NRF_RADIO_BLE_CODED
 	select HAS_HW_NRF_RADIO_IEEE802154
+	select HAS_HW_NRF_RADIO_BLE_DF
```

Comment:
I will add select for other SOCs in future after making tests if everything works there also. Currently I use only nRF52833 and that is the cause I selected it only for that SOC.

Reply:
Ok, that's my misunderstanding then. :) Since it was nrf52/Kconfig.soc and I didn't look at the complete context I mistakenly thought it had been added to all SoCs in nrf52 series.